tools/crm_resource_runtime.c

```diff
@@ -188,18 +189,21 @@ find_matching_attr_resource(resource_t * rsc, const char * rsc_id, const char *
                 break;
         }
 
-    } else if (rsc->parent && BE_QUIET == FALSE) {
+    } else if(rsc->parent && BE_QUIET == FALSE) {
         printf("Forcing %s of '%s' for '%s' instead of '%s'\n", cmd, attr_name, rsc_id, rsc->parent->id);
 
-    } else if(rsc->parent == NULL && rsc->children) {
-        resource_t *child = rsc->children->data;
+    } else if(rsc->parent == NULL) {
```

Comment:
rsc->parent is always NULL at this point, so "else" is sufficient

Reply:
Done.